mediacapture-streams/MediaStream-add-audio-track.https.html

```diff
@@ -15,7 +15,7 @@ <h1 class="instructions">Description</h1>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script>
-var t = async_test("Tests that adding a track to a MediaStream works as expected", {timeout: 20000}); // longer timeout since requires double user interaction
+var t = async_test("Tests that adding a track to a MediaStream works as expected"); // longer timeout since requires double user interaction
```

Comment:
Can you remove the comment which refers to the deleted code?

Reply:
Oops! Most of the update was done by a script and I didn't notice that unless comment. It's removed now.